library/SimplePie/Item.php

```diff
@@ -90,7 +90,26 @@ class SimplePie_Item
 	public function __construct($feed, $data)
 	{
 		$this->feed = $feed;
-		$this->data = $data;
+		$this->data = $this->array_map_recursive('trim', $data);
```

Comment:
I should've mentioned that too. The constructor is the wrong place to be manipulating data like this. It doesn't logically make sense to say that data should be trimmed here, as there is nothing inherently wrong with having white space.